docs/component/open-attestation.md

```diff
@@ -4,8 +4,311 @@ title: OpenAttestation Wrapper SDK (javascript)
+        identityProof: {
+          type: '5fc379dd-cd24-4c91-a4c7-cf76f9c96d8d:string:DNS-TXT',
+          location: '62e1c6c3-09a9-4e52-aaf6-9daf24d43657:string:example.com'
+        }
+      }
+    ]
+  },
+  privacy: { obfuscatedData: [] },
+  signature: {
+    type: 'SHA3MerkleProof',
+    targetHash: '956e27c86d4893a971600d328e235ede886f1c00b183257a822667b69886fced',
+    proof: [],
+    merkleRoot: '956e27c86d4893a971600d328e235ede886f1c00b183257a822667b69886fced'
+  }
+}
+```
+Note: 
+- Every time you run `wrapDocument`, it will create different hashes (in front of every field in the data object), `targetHash` and `merkleRoot`.
+- `targetHash` and `merkleRoot` are always the same when wrapping one document.
+
+#### Wrapping multiple documents
+```javascript
+const { wrapDocuments } = require("@govtechsg/open-attestation");
+const document = require("./document.json");
+const util = require("util");
+
+const wrappedDocuments = wrapDocuments([document, document]);
+
+console.log(util.inspect(wrappedDocuments, { showHidden: false, depth: null }));
+```
+ This will display a similar result as previously with few differences:
+- `wrapDocuments` returns an `array` and not an `object`.
+- Each element in the array is a wrapped document corresponding to the one provided as input.
+- Each element has the same `merkleRoot`.
+- Each element has a different `targetHash`.
+- Similar to `wrapDocument`, every time you run `wrapDocuments`, it will create different hashes (in front of every fields in the data object), `targetHash` and `merkleRoot`.
+ 
+#### Wrapping invalid document
+Every document that you wrap must conform to the [OA schema](https://raw.githubusercontent.com/Open-Attestation/open-attestation/master/src/schema/2.0/schema.json). If that's not the case, an error will be thrown providing information about why your document is not valid.
+
+For instance let's change `document.json` to remove the `issuers` field so that the document is an empty object:
+```json
+{}
+```
+
+Let's wrap again this document:
+```javascript
+const { wrapDocument } = require("@govtechsg/open-attestation");
+const document = require("./document.json");
+const util = require("util");
+
+const wrappedDocument = wrapDocument(document);
+
+console.log(util.inspect(wrappedDocument, { showHidden: false, depth: null }));
+```
+
+Running the code will output that the document is not valid, because the issuers property is required:
+```javascript
+Error: Invalid document
+    at new SchemaValidationError
+...
+validationErrors: [
+    {
+      keyword: 'required',
+      dataPath: '',
+      schemaPath: '#/required',
+      params: [Object],
+      message: "should have required property 'issuers'"
+    }
+  ],
+```
+
+#### Adding data
+You are free to add the data you want in the document. OA only cares about the data it needs to work and doesn't perform extra verification on the additional data you provide.
```

Comment:
```suggestion
You are free to add additional data you want in the document. OA only cares about the data it needs to work and doesn't perform extra verification on the additional data you provide.
```

Shall we describe what we mean by 'OA only cares about the data it needs'? What does OA actually need?